components/base/kinematicbase/ptgKinematics.go

```diff
@@ -36,7 +36,6 @@ type ptgBaseKinematics struct {
 	motion.Localizer
 	logger       golog.Logger
 	frame        referenceframe.Frame
-	fs           referenceframe.FrameSystem
```

Comment:
removing this since we do not use the `fs` field at all.
It also seems a little confusing to a separate fs constructed as the motion service level and also have this one.

Reply:
Agree. I went back and looked through the commits where I added that and could not figure out why I put it in.